great_expectations/checkpoint/actions.py

```diff
@@ -182,10 +182,10 @@ def __new__(cls, clsname, bases, attrs):
 @public_api
 class ValidationAction(BaseModel, metaclass=MetaValidationAction):
     """
-    ValidationActions define a set of steps to be run after a validation result is produced.
+    Actions define a set of steps to run after a Validation Result is produced. Subclass `ValidationAction` to create a `custom Action </docs/core/trigger_actions_based_on_results/create_a_custom_action>`_.
```

Comment:
**Note for reviewer**: though I've written "custom" in lowercase here, it's getting capitalized in the output
<img width="1392" alt="Screenshot 2024-12-13 at 1 10 03 PM" src="https://github.com/user-attachments/assets/b637baee-a8e8-4da8-9d7e-0856eec7d1f9" />
is there any way I can force "custom" to display as lowercase here? this is my first time editing a docstring for our API reference

Reply:
Huh that is a little odd - I don't know the exact logic that does this but it might be ingrained in one of the docs libs we use.

Perhaps try MD syntax for hyperlinks? Not sure that's supported but worth a shot?

Reply:
MD syntax worked! 🎉